src/ledger/LedgerManagerImpl.h

```diff
@@ -86,9 +86,7 @@ class LedgerManagerImpl : public LedgerManager
     uint64_t secondsSinceLastLedgerClose() const override;
     void syncMetrics() override;
 
-    void startNewLedger(uint32_t protocolVersion, int64_t balance,
-                        uint32_t baseFee, uint32_t baseReserve,
-                        uint32_t maxTxSetSize);
+    void startNewLedger(LedgerHeader genesisLedger);
```

Comment:
use a `const&` as there is no reason to copy this object around

Reply:
Ledger delta constructor requires non-const LedgerHeader and it is used in startNewLedger